ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/MPCEstimatorTask.java

```diff
@@ -0,0 +1,99 @@
+package us.ihmc.avatar;
+
+import us.ihmc.avatar.factory.MPCHumanoidRobotControlTask;
+import us.ihmc.commonWalkingControlModules.barrierScheduler.context.HumanoidRobotMPCContextData;
+import us.ihmc.commonWalkingControlModules.controllerCore.command.MPCCrossRobotCommandResolver;
+import us.ihmc.robotModels.FullHumanoidRobotModel;
+import us.ihmc.robotics.time.ThreadTimer;
+import us.ihmc.yoVariables.variable.YoLong;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MPCEstimatorTask extends MPCHumanoidRobotControlTask
```

Comment:
Is there any difference between this and the regular one? I'd use the regular one if there is not.

Reply:
The updateMasterContext and updateLocalContext use different type of input ("HumaonidRobotMPCContextData")